x/pointer.go

```diff
@@ -11,10 +11,3 @@ func PointToUUID(id uuid.UUID) *uuid.UUID {
 	}
 	return &id
 }
```

Comment:
You should be able to remove the `PointToUUID` function here as well.

Reply:
`PointToUUID` has different semantics than `pointerx.Ptr`.